websites/K/Kinopub/presence.ts

```diff
@@ -0,0 +1,254 @@
+				presenceData.details = `${Strings.Read} ${Strings.Post}`;
+				presenceData.state = textContent(".text-success");
+				presenceData.smallImageKey = Assets.Read;
+				presenceData.smallImageText = Strings.Read;
+			}
+			break;
+
+		case "plugin":
+			presenceData.details = `${Strings.Read} ${Strings.Instructions}`;
+			presenceData.smallImageKey = Assets.Read;
+			presenceData.smallImageText = Strings.Read;
+			break;
+
+		case "users":
+			presenceData.details = `${Strings.View} ${Strings.Page} ${
+				!privacy ? textContent(".page-title span") : Strings.User
+			}`;
+			presenceData.state = textContent(".nav-link.active");
+			presenceData.smallImageKey = Assets.View;
+			presenceData.smallImageText = Strings.View;
+			break;
+
+		case "user":
+			presenceData.details = Strings.Settings;
+			presenceData.state = textContent(".nav-item.active");
+			presenceData.smallImageKey = Assets.View;
+			presenceData.smallImageText = Strings.View;
+			break;
+
+		case "notification":
+			presenceData.details = `${Strings.View} ${Strings.Notifications}`;
+			presenceData.state = textContent(".nav-link.active");
+			presenceData.smallImageKey = Assets.View;
+			presenceData.smallImageText = Strings.View;
+			break;
+
+		case "item":
+			if (pathname.split("/")[2] === "search") {
+				presenceData.details = `${Strings.Search}: ${
+					document.querySelector<HTMLInputElement>('input[name="query"]').value
+				}`;
+				presenceData.smallImageKey = Assets.Search;
+				presenceData.smallImageText = Strings.Search;
+			} else {
+				presenceData.details = `${Strings.View} ${Strings.Page} ${
+					serieName ? Strings.Serial : Strings.Movie
+				}`;
+				presenceData.state =
+					document.querySelector(
+						".page-content h3"
+					)?.childNodes[0]?.textContent;
+				if (!privacy && cover) {
+					presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+						".item-poster-relative"
+					).src;
+				}
+				presenceData.smallImageKey = Assets.View;
+				presenceData.smallImageText = Strings.View;
+				presenceData.buttons = [
+					{
+						label: Strings.OpenLink,
+						url: href,
+					},
+				];
+
+				if (!privacy && video.duration) {
+					presenceData.details =
+						document.querySelector(
+							".page-content h3"
+						)?.childNodes[0]?.textContent;
+					presenceData.state = serieName;
+					presenceData.smallImageKey = video.paused
+						? Assets.Pause
+						: Assets.Play;
+					presenceData.smallImageText = video.paused
+						? Strings.Pause
+						: Strings.Play;
+
+					if (video.paused || !time) {
+						delete presenceData.startTimestamp;
+						delete presenceData.endTimestamp;
+					} else {
+						[presenceData.startTimestamp, presenceData.endTimestamp] =
+							presence.getTimestamps(video.currentTime, video.duration);
+					}
+				}
+			}
+			break;
+	}
+
+	if (privacy) delete presenceData.state;
+	if (privacy || !buttons) delete presenceData.buttons;
+	presence.setActivity(presenceData);
```

Comment:
This suggestion is optional, but this makes it just show the logo if it errors or sum.
```suggestion
	if (presenceData.details) presence.setActivity(presenceData);
	else presence.setActivity();
```

Reply:
> This suggestion is optional, but this makes it just show the logo if it errors or sum.

By default, if the path is not found, "Somewhere on the site" is displayed.